crates/compiler/build/src/program.rs

```diff
@@ -166,12 +166,11 @@ fn gen_from_mono_module_llvm<'a>(
     let context = Context::create();
     let module = arena.alloc(module_from_builtins(target, &context, "app"));
 
-    // mark our zig-defined builtins as internal
     let app_ll_file = {
-        let mut temp = PathBuf::from(roc_file_path);
-        temp.set_extension("ll");
+        let mut roc_file_path_buf = PathBuf::from(roc_file_path);
+        roc_file_path_buf.set_extension("ll");
 
-        temp
+        roc_file_path_buf
```

Comment:
This variable is temporary but the `app_ll_file` is not, so I changed it to avoid confusion.